webpack/ForemanTasks/Components/TasksTable/Components/ConfirmModal/ConfirmModal.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { startCase } from 'lodash';
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import { Button } from 'patternfly-react';
+import ForemanModal from 'foremanReact/components/ForemanModal';
+import { useForemanModal } from 'foremanReact/components/ForemanModal/ForemanModalHooks';
+import { getURIQuery } from 'foremanReact/common/helpers';
+
+export const ConfirmModal = ({
+  actionText,
+  actionState,
+  actionType,
+  selectedRowsLen,
+  id,
+  parentTaskID,
+  url,
+  ...tasksActions
+}) => {
+  const query = getURIQuery(url);
+  const { setModalClosed } = useForemanModal({
+    id,
+  });
```

Comment:
the parent already uses the `useForemanModal` hook for an id which equals also to CONFIRM_MODAL,
can you pass it from there?